internal/pkg/client/oci/oci.go

```diff
@@ -114,11 +115,16 @@ func ImageSHA(ctx context.Context, uri string, sys *types.SystemContext) (string
 	return calculateRefHash(ctx, ref, sys)
 }
 
-func calculateRefHash(ctx context.Context, ref types.ImageReference, sys *types.SystemContext) (string, error) {
+func calculateRefHash(ctx context.Context, ref types.ImageReference, sys *types.SystemContext) (retStr string, retErr error) {
```

Comment:
Please could you use `err` as we are using elsewhere in the code.

Also, instead of retStr the return can be called `hash` as it is in the function body.

Reply:
Sure!